engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/command/InterruptContextCmd.scala

```diff
@@ -23,11 +23,7 @@ class InterruptContextCmd(
   ): Future[Unit] =
     if (doesContextExist) {
       Future {
-        ctx.jobControlPlane.abortJobs(
-          request.contextId,
-          "interrupt context",
-          false
-        )
+        ctx.executionService.getContext.getThreadManager.interruptThreads()
```

Comment:
but `abortJobs` also calls `getContext.getThreadManager.interruptThreads()`. I don't understand the change